### PR TITLE
Update the site to reflect the first release

### PR DESCRIPTION
**Pull request by davorbonaci**
- Add a blog post about the first release.
- Add a page about releases.
- Add Javadoc for the first release (still TODO).
- Rebuild the site, that includes publishing of several unpublished changes.

That said, the rebuild still has suspicious changes that need a triage. And, it is pretty crude.

R: @evilsoapbox, @francesperry.

Staged at: http://172.31.90.60:4000/

### CODE REVIEW

**Comment by evilsoapbox** [comment]
Overall looks good; yay! A few minor suggestions (which are not blocking):

* What about a new top-level tab for "Downloads" and we can have a section in there for the latest stable release (which points to /releases) and a section in there for a less stable (nightly) release?
* For the Javadocs - what about a navigation scheme where we have a section called "Release documentation" and then re-label /Java API Reference/0.0.1-incubating Javadocs/?

**Comment by davorbonaci** [comment]
All great ideas -- I'd leave them for a followup change and/or a wider re-organization.

**Comment by evilsoapbox** [comment]
SGTM - looks all good to me then!

File: _pages/releases.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+title: "Apache Beam Releases"
+permalink: /releases/
+---
+
+# Apache Beam releases
+
+The easiest way to use Apache Beam is via one of the released versions in the
+[Maven Central Repository](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.apache.beam%22).
+
+For example, if you are developing using Maven, add the following dependencies
```
Comment (francesperry) [inline _pages/releases.md]: if you are developing using Maven and want to use the Java SDK with the DirectPipelineRunner, add...

(remove such as)

File: _pages/releases.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Additionally, you may want to depend on additional SDK modules, such as IO
+connectors or other extensions, and additional runners to execute your pipeline
+at scale.
+
+## Release Notes
+
+### 0.1.0-incubating
+[Source code download](https://dist.apache.org/repos/dist/release/incubator/beam/0.1.0-incubating/apache-beam-0.1.0-incubating-source-release.zip)
+
+* The first incubating release of Apache Beam.
```
Comment (francesperry) [inline _pages/releases.md]: Add caveat about being a process-focused release?

**Comment by francesperry** [comment]
LGTM. 

Agree there are some larger questions around navigation but ok to follow up on those next.

File: _pages/releases.md (hunk 2 above)
Comment (davorbonaci) [inline _pages/releases.md]: I'd leave it out.

File: _pages/releases.md (hunk 1 above)
Comment (davorbonaci) [inline _pages/releases.md]: Done.

**Comment by davorbonaci** [comment]
Updated with Javadoc, etc.

Final call for comments.

**Pull request merged**